packages/gamut-styles/src/variables/colors.ts

```diff
@@ -3,6 +3,9 @@ const white = '#ffffff';
 
 /**
  * @deprecated
+ * Using these variables directly is no longer supported.
+ *
+ * Please use [`theme.colors`](https://gamut.codecademy.com/storybook/?path=/docs/foundations-theme--colors#standard-colors)
```

Comment:
Awesome, TIL markdown links work in JSDocs now!

```suggestion
 * Please use [`theme.colors`](https://gamut.codecademy.com/storybook/?path=/docs/foundations-theme--colors#color)
```

Reply:
Heh yeah its the workaround for TS and @link not working apparently.